docs/reduction-in-locks.md

```diff
@@ -12,17 +12,17 @@
 
 The `--lock-ddl=REDUCED` option features are as follows:
 
-* Acquire and release the backup lock quickly: The backup process minimizes the time it holds the backup lock, allowing for concurrent DDL operations.
+* Acquire and release the backup lock quickly: The backup process minimizes the time the server holds the backup lock, allowing for concurrent DDL operations.
 
-    ??? note "Comparing the backup duration with --lock-ddl=REDUCED"
+    ??? note "Comparing the server lock duration with --lock-ddl=REDUCED"
 
         While we strive to provide accurate performance benchmarks, real-world results may vary depending on your hardware and software configurations.
 
         The following tables illustrate the approximate time differences between `--lock-ddl=ON` and `--lock-ddl=REDUCED` for local and cloud backups (Amazon S3). The `Improvement (X times)` column shows how many times less the server is locked by xtrabackup using  `--lock-ddl=REDUCED` compared to `--lock-ddl=ON`.
 
         === "Local backup"
 
-	        | Data size in gigabytes | Backup duration with --lock-ddl=ON in seconds| Backup duration with --lock-ddl=REDUCED in seconds| Improvement (X times)  |
+	        | Data size in gigabytes | Server lock duration with --lock-ddl=ON in seconds| Server lock duration with --lock-ddl=REDUCED in seconds| Improvement (X times)  |
```

Comment:
May be lets use "backup lock duration". I see that this is the title :) "Reduced backup lock time"